demo/src/examples/examples-page.js

```diff
@@ -23,45 +23,22 @@ export class ExamplesPage extends LitElement {
       }`
   ];
 
-  #openExample(event) {
-    const exampleEl = event.target.closest('button');
-
-    // Check if the clicked element is a button and has the 'section' dataset attribute.
-    if (!exampleEl || !('section' in exampleEl.parentNode.dataset)) return;
-
-
-    const parent = exampleEl.parentNode;
-    const section = parent.dataset.section;
-    const exampleIdx = Array.from(parent.children)
-      .filter(child => child.tagName.toLowerCase() === 'button')
-      .indexOf(exampleEl);
-
-    openPlayerModal(Examples[section][exampleIdx]);
-  }
-
   render() {
     return html`
       <load-media-form @submit-media="${e => openPlayerModal(e.detail)}">
       </load-media-form>
 
       <!-- List of examples -->
       <div class="fade-in"
-           @animationend="${e => e.target.classList.remove('fade-in')}"
-           @click="${this.#openExample.bind(this)}">
+           @animationend="${e => e.target.classList.remove('fade-in')}">
         ${map(Object.entries(Examples), ([section, examples]) => html`
           <div class="example-section" data-section="${section}">
             <h2 class="sticky">${section}</h2>
             ${map(examples, example => html`
-              <button class="content-btn"
-                      title="${example.description || example.title}">
-                <span class="content-btn-title">
-                  ${example.description || example.title}
-                </span>
-                ${when(example.description, () => html`
-                  <div class="content-btn-metadata-container">
-                    <span class="content-btn-info">${example.title}</span>
-                  </div>`)}
-              </button>
+              <content-link title="${example.description || example.title}"
+                            href="examples?${asQueryParams(example)}">
+                ${example.description ? example.title : ''}
```

Comment:
the description isn't rendered without something like
```suggestion
   <span slot="description">
                 ${example.description ? example.title : ''}
   </span>
```